nilearn/glm/__init__.py

```diff
@@ -47,3 +47,8 @@
     'first_level',
     'second_level',
 ]
+
+from warnings import warn
+
+warn('This module is experimental. ' +
```

Comment:
It is probably useful to indicate which model you are talking about: in a cell with many module imports, the user will see the warning pop up not knowing which import triggered it.